<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />

<meta name="viewport" content="width=device-width, initial-scale=1" />

<meta name="author" content="Tony Long" />

<meta name="date" content="2025-06-28" />

<title>Using XQTL2.Xplore</title>

<script>// Pandoc 2.9 adds attributes on both header and div. We remove the former (to
// be compatible with the behavior of Pandoc < 2.8).
document.addEventListener('DOMContentLoaded', function(e) {
  var hs = document.querySelectorAll("div.section[class*='level'] > :first-child");
  var i, h, a;
  for (i = 0; i < hs.length; i++) {
    h = hs[i];
    if (!/^h[1-6]$/i.test(h.tagName)) continue;  // it should be a header h1-h6
    a = h.attributes;
    while (a.length > 0) h.removeAttribute(a[0].name);
  }
});
</script>

<style type="text/css">
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
</style>



<style type="text/css">
code {
white-space: pre;
}
.sourceCode {
overflow: visible;
}
</style>
<style type="text/css" data-origin="pandoc">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
{ counter-reset: source-line 0; }
pre.numberSource code > span
{ position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
{ content: counter(source-line);
position: relative; left: -1em; text-align: right; vertical-align: baseline;
border: none; display: inline-block;
-webkit-touch-callout: none; -webkit-user-select: none;
-khtml-user-select: none; -moz-user-select: none;
-ms-user-select: none; user-select: none;
padding: 0 4px; width: 4em;
color: #aaaaaa;
}
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa; padding-left: 4px; }
div.sourceCode
{ }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } 
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } 
code span.at { color: #7d9029; } 
code span.bn { color: #40a070; } 
code span.bu { color: #008000; } 
code span.cf { color: #007020; font-weight: bold; } 
code span.ch { color: #4070a0; } 
code span.cn { color: #880000; } 
code span.co { color: #60a0b0; font-style: italic; } 
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } 
code span.do { color: #ba2121; font-style: italic; } 
code span.dt { color: #902000; } 
code span.dv { color: #40a070; } 
code span.er { color: #ff0000; font-weight: bold; } 
code span.ex { } 
code span.fl { color: #40a070; } 
code span.fu { color: #06287e; } 
code span.im { color: #008000; font-weight: bold; } 
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } 
code span.kw { color: #007020; font-weight: bold; } 
code span.op { color: #666666; } 
code span.ot { color: #007020; } 
code span.pp { color: #bc7a00; } 
code span.sc { color: #4070a0; } 
code span.ss { color: #bb6688; } 
code span.st { color: #4070a0; } 
code span.va { color: #19177c; } 
code span.vs { color: #4070a0; } 
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } 
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    var j = 0;
    while (j < rules.length) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") {
        j++;
        continue;
      }
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') {
        j++;
        continue;
      }
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>




<style type="text/css">body {
background-color: #fff;
margin: 1em auto;
max-width: 700px;
overflow: visible;
padding-left: 2em;
padding-right: 2em;
font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
font-size: 14px;
line-height: 1.35;
}
#TOC {
clear: both;
margin: 0 0 10px 10px;
padding: 4px;
width: 400px;
border: 1px solid #CCCCCC;
border-radius: 5px;
background-color: #f6f6f6;
font-size: 13px;
line-height: 1.3;
}
#TOC .toctitle {
font-weight: bold;
font-size: 15px;
margin-left: 5px;
}
#TOC ul {
padding-left: 40px;
margin-left: -1.5em;
margin-top: 5px;
margin-bottom: 5px;
}
#TOC ul ul {
margin-left: -2em;
}
#TOC li {
line-height: 16px;
}
table {
margin: 1em auto;
border-width: 1px;
border-color: #DDDDDD;
border-style: outset;
border-collapse: collapse;
}
table th {
border-width: 2px;
padding: 5px;
border-style: inset;
}
table td {
border-width: 1px;
border-style: inset;
line-height: 18px;
padding: 5px 5px;
}
table, table th, table td {
border-left-style: none;
border-right-style: none;
}
table thead, table tr.even {
background-color: #f7f7f7;
}
p {
margin: 0.5em 0;
}
blockquote {
background-color: #f6f6f6;
padding: 0.25em 0.75em;
}
hr {
border-style: solid;
border: none;
border-top: 1px solid #777;
margin: 28px 0;
}
dl {
margin-left: 0;
}
dl dd {
margin-bottom: 13px;
margin-left: 13px;
}
dl dt {
font-weight: bold;
}
ul {
margin-top: 0;
}
ul li {
list-style: circle outside;
}
ul ul {
margin-bottom: 0;
}
pre, code {
background-color: #f7f7f7;
border-radius: 3px;
color: #333;
white-space: pre-wrap; 
}
pre {
border-radius: 3px;
margin: 5px 0px 10px 0px;
padding: 10px;
}
pre:not([class]) {
background-color: #f7f7f7;
}
code {
font-family: Consolas, Monaco, 'Courier New', monospace;
font-size: 85%;
}
p > code, li > code {
padding: 2px 0px;
}
div.figure {
text-align: center;
}
img {
background-color: #FFFFFF;
padding: 2px;
border: 1px solid #DDDDDD;
border-radius: 3px;
border: 1px solid #CCCCCC;
margin: 0 5px;
}
h1 {
margin-top: 0;
font-size: 35px;
line-height: 40px;
}
h2 {
border-bottom: 4px solid #f7f7f7;
padding-top: 10px;
padding-bottom: 2px;
font-size: 145%;
}
h3 {
border-bottom: 2px solid #f7f7f7;
padding-top: 10px;
font-size: 120%;
}
h4 {
border-bottom: 1px solid #f7f7f7;
margin-left: 8px;
font-size: 105%;
}
h5, h6 {
border-bottom: 1px solid #ccc;
font-size: 105%;
}
a {
color: #0033dd;
text-decoration: none;
}
a:hover {
color: #6666ff; }
a:visited {
color: #800080; }
a:visited:hover {
color: #BB00BB; }
a[href^="http:"] {
text-decoration: underline; }
a[href^="https:"] {
text-decoration: underline; }

code > span.kw { color: #555; font-weight: bold; } 
code > span.dt { color: #902000; } 
code > span.dv { color: #40a070; } 
code > span.bn { color: #d14; } 
code > span.fl { color: #d14; } 
code > span.ch { color: #d14; } 
code > span.st { color: #d14; } 
code > span.co { color: #888888; font-style: italic; } 
code > span.ot { color: #007020; } 
code > span.al { color: #ff0000; font-weight: bold; } 
code > span.fu { color: #900; font-weight: bold; } 
code > span.er { color: #a61717; background-color: #e3d2d2; } 
</style>




</head>

<body>




<h1 class="title toc-ignore">Using XQTL2.Xplore</h1>
<h4 class="author">Tony Long</h4>
<h4 class="date">2025-06-28</h4>



<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>XQTL2.Xplore provides tools for visualizing XQTL (X-chromosome
Quantitative Trait Locus) data. This vignette demonstrates how to use
the packageâ€™s main functions.</p>
</div>
<div id="data-objects" class="section level1">
<h1>Data Objects</h1>
<p>The package includes several pre-processed data objects:</p>
<div id="reference-data" class="section level2">
<h2>Reference Data</h2>
<ol style="list-style-type: decimal">
<li><code>dm6.ncbiRefSeq.genes</code>: Processed gene annotations from
the dm6 reference genome</li>
<li><code>dm6.variants</code>: Processed variant data from multiple VCF
files</li>
</ol>
</div>
<div id="example-datasets" class="section level2">
<h2>Example Datasets</h2>
<ol start="3" style="list-style-type: decimal">
<li><code>zinc_hanson_pseudoscan</code>: QTL scan results from ZINC
Hanson experiment</li>
<li><code>zinc_hanson_means</code>: Founder frequency data from ZINC
Hanson experiment</li>
</ol>
</div>
<div id="updating-data-objects" class="section level2">
<h2>Updating Data Objects</h2>
<p>The data objects can be updated independently using the functions in
the <code>data-raw</code> directory:</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" tabindex="-1"></a><span class="co"># To update the genes data (from GTF file)</span></span>
<span id="cb1-2"><a href="#cb1-2" tabindex="-1"></a><span class="fu">source</span>(<span class="st">&quot;data-raw/create_genes_object.R&quot;</span>)</span>
<span id="cb1-3"><a href="#cb1-3" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">document</span>()</span>
<span id="cb1-4"><a href="#cb1-4" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">install</span>()</span>
<span id="cb1-5"><a href="#cb1-5" tabindex="-1"></a><span class="fu">data</span>(dm6.ncbiRefSeq.genes)</span>
<span id="cb1-6"><a href="#cb1-6" tabindex="-1"></a></span>
<span id="cb1-7"><a href="#cb1-7" tabindex="-1"></a><span class="co"># To update the variants data (from VCF files)</span></span>
<span id="cb1-8"><a href="#cb1-8" tabindex="-1"></a><span class="fu">source</span>(<span class="st">&quot;data-raw/create_variants_object.R&quot;</span>)</span>
<span id="cb1-9"><a href="#cb1-9" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">document</span>()</span>
<span id="cb1-10"><a href="#cb1-10" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">install</span>()</span>
<span id="cb1-11"><a href="#cb1-11" tabindex="-1"></a><span class="fu">data</span>(dm6.variants)</span>
<span id="cb1-12"><a href="#cb1-12" tabindex="-1"></a></span>
<span id="cb1-13"><a href="#cb1-13" tabindex="-1"></a><span class="co"># To update example datasets</span></span>
<span id="cb1-14"><a href="#cb1-14" tabindex="-1"></a><span class="fu">source</span>(<span class="st">&quot;data-raw/create_example_data.R&quot;</span>)</span>
<span id="cb1-15"><a href="#cb1-15" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">document</span>()</span>
<span id="cb1-16"><a href="#cb1-16" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">install</span>()</span>
<span id="cb1-17"><a href="#cb1-17" tabindex="-1"></a><span class="fu">data</span>(zinc_hanson_pseudoscan)</span>
<span id="cb1-18"><a href="#cb1-18" tabindex="-1"></a><span class="fu">data</span>(zinc_hanson_means)</span></code></pre></div>
<p>Note: The VCF processing is computationally intensive, so only run
<code>create_variants_object.R</code> when necessary.</p>
</div>
</div>
<div id="basic-usage" class="section level1">
<h1>Basic Usage</h1>
<div id="loading-package-data" class="section level2">
<h2>Loading Package Data</h2>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" tabindex="-1"></a><span class="fu">library</span>(XQTL2.Xplore)</span>
<span id="cb2-2"><a href="#cb2-2" tabindex="-1"></a></span>
<span id="cb2-3"><a href="#cb2-3" tabindex="-1"></a><span class="co"># Load reference data</span></span>
<span id="cb2-4"><a href="#cb2-4" tabindex="-1"></a><span class="fu">data</span>(dm6.ncbiRefSeq.genes)</span>
<span id="cb2-5"><a href="#cb2-5" tabindex="-1"></a><span class="fu">data</span>(dm6.variants)</span>
<span id="cb2-6"><a href="#cb2-6" tabindex="-1"></a></span>
<span id="cb2-7"><a href="#cb2-7" tabindex="-1"></a><span class="co"># Load example datasets</span></span>
<span id="cb2-8"><a href="#cb2-8" tabindex="-1"></a><span class="fu">data</span>(zinc_hanson_pseudoscan)</span>
<span id="cb2-9"><a href="#cb2-9" tabindex="-1"></a><span class="fu">data</span>(zinc_hanson_means)</span></code></pre></div>
</div>
<div id="loading-your-own-data" class="section level2">
<h2>Loading Your Own Data</h2>
<p>If you have your own data, you can load it using standard R
functions:</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" tabindex="-1"></a><span class="co"># Load your own QTL scan results</span></span>
<span id="cb3-2"><a href="#cb3-2" tabindex="-1"></a>my_qtl_data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">&quot;my_pseudoscan.txt&quot;</span>, <span class="at">header =</span> <span class="cn">TRUE</span>)</span>
<span id="cb3-3"><a href="#cb3-3" tabindex="-1"></a></span>
<span id="cb3-4"><a href="#cb3-4" tabindex="-1"></a><span class="co"># Load your own frequency data</span></span>
<span id="cb3-5"><a href="#cb3-5" tabindex="-1"></a>my_freq_data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">&quot;my_meansBySample.txt&quot;</span>, <span class="at">header =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</div>
<div id="creating-plots" class="section level2">
<h2>Creating Plots</h2>
<div id="manhattan-plot" class="section level3">
<h3>Manhattan Plot</h3>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" tabindex="-1"></a><span class="co"># Using example data</span></span>
<span id="cb4-2"><a href="#cb4-2" tabindex="-1"></a><span class="fu">XQTL_Manhattan_5panel</span>(zinc_hanson_pseudoscan)</span>
<span id="cb4-3"><a href="#cb4-3" tabindex="-1"></a></span>
<span id="cb4-4"><a href="#cb4-4" tabindex="-1"></a><span class="co"># Using your own data</span></span>
<span id="cb4-5"><a href="#cb4-5" tabindex="-1"></a><span class="fu">XQTL_Manhattan_5panel</span>(my_qtl_data)</span>
<span id="cb4-6"><a href="#cb4-6" tabindex="-1"></a></span>
<span id="cb4-7"><a href="#cb4-7" tabindex="-1"></a><span class="co"># Single Manhattan plot</span></span>
<span id="cb4-8"><a href="#cb4-8" tabindex="-1"></a><span class="fu">XQTL_Manhattan</span>(zinc_hanson_pseudoscan)</span></code></pre></div>
</div>
<div id="region-plot" class="section level3">
<h3>Region Plot</h3>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" tabindex="-1"></a><span class="co"># Using example data</span></span>
<span id="cb5-2"><a href="#cb5-2" tabindex="-1"></a>A1 <span class="ot">=</span> <span class="fu">XQTL_region</span>(zinc_hanson_pseudoscan, <span class="st">&quot;chr3R&quot;</span>, <span class="dv">13000000</span>, <span class="dv">15000000</span>, <span class="st">&quot;Wald_log10p&quot;</span>)</span>
<span id="cb5-3"><a href="#cb5-3" tabindex="-1"></a>A2 <span class="ot">=</span> <span class="fu">XQTL_change_average</span>(zinc_hanson_means, <span class="st">&quot;chr3R&quot;</span>, <span class="dv">13000000</span>, <span class="dv">15000000</span>)</span>
<span id="cb5-4"><a href="#cb5-4" tabindex="-1"></a>A3 <span class="ot">=</span> <span class="fu">XQTL_genes</span>(dm6.ncbiRefSeq.genes, <span class="st">&quot;chr3R&quot;</span>, <span class="dv">13000000</span>, <span class="dv">15000000</span>)</span>
<span id="cb5-5"><a href="#cb5-5" tabindex="-1"></a></span>
<span id="cb5-6"><a href="#cb5-6" tabindex="-1"></a><span class="co"># Using your own data</span></span>
<span id="cb5-7"><a href="#cb5-7" tabindex="-1"></a>A1 <span class="ot">=</span> <span class="fu">XQTL_region</span>(my_qtl_data, <span class="st">&quot;chr3R&quot;</span>, <span class="dv">13000000</span>, <span class="dv">15000000</span>, <span class="st">&quot;Wald_log10p&quot;</span>)</span>
<span id="cb5-8"><a href="#cb5-8" tabindex="-1"></a>A2 <span class="ot">=</span> <span class="fu">XQTL_change_average</span>(my_freq_data, <span class="st">&quot;chr3R&quot;</span>, <span class="dv">13000000</span>, <span class="dv">15000000</span>)</span>
<span id="cb5-9"><a href="#cb5-9" tabindex="-1"></a>A3 <span class="ot">=</span> <span class="fu">XQTL_genes</span>(dm6.ncbiRefSeq.genes, <span class="st">&quot;chr3R&quot;</span>, <span class="dv">13000000</span>, <span class="dv">15000000</span>)</span></code></pre></div>
</div>
<div id="finding-peaks" class="section level3">
<h3>Finding Peaks</h3>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" tabindex="-1"></a><span class="co"># Using example data</span></span>
<span id="cb6-2"><a href="#cb6-2" tabindex="-1"></a>out <span class="ot">=</span> <span class="fu">XQTL_zoom</span>(zinc_hanson_pseudoscan, <span class="st">&quot;chr3R&quot;</span>, <span class="dv">13000000</span>, <span class="dv">15000000</span>, <span class="at">left_drop =</span> <span class="dv">3</span>, <span class="at">right_drop =</span> <span class="dv">3</span>)</span>
<span id="cb6-3"><a href="#cb6-3" tabindex="-1"></a></span>
<span id="cb6-4"><a href="#cb6-4" tabindex="-1"></a><span class="co"># Using your own data</span></span>
<span id="cb6-5"><a href="#cb6-5" tabindex="-1"></a>out <span class="ot">=</span> <span class="fu">XQTL_zoom</span>(my_qtl_data, <span class="st">&quot;chr3R&quot;</span>, <span class="dv">13000000</span>, <span class="dv">15000000</span>, <span class="at">left_drop =</span> <span class="dv">3</span>, <span class="at">right_drop =</span> <span class="dv">3</span>)</span></code></pre></div>
</div>
</div>
</div>
<div id="advanced-usage" class="section level1">
<h1>Advanced Usage</h1>
<div id="customizing-plots" class="section level2">
<h2>Customizing Plots</h2>
<p>All plotting functions use ggplot2 and can be customized using
standard ggplot2 syntax:</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" tabindex="-1"></a><span class="co"># Example: Customizing a Manhattan plot</span></span>
<span id="cb7-2"><a href="#cb7-2" tabindex="-1"></a>p <span class="ot">=</span> <span class="fu">XQTL_Manhattan</span>(zinc_hanson_pseudoscan)</span>
<span id="cb7-3"><a href="#cb7-3" tabindex="-1"></a>p <span class="sc">+</span> <span class="fu">theme_minimal</span>() <span class="sc">+</span> </span>
<span id="cb7-4"><a href="#cb7-4" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">&quot;My Custom Title&quot;</span>)</span></code></pre></div>
</div>
<div id="working-with-large-regions" class="section level2">
<h2>Working with Large Regions</h2>
<p>For large genomic regions, consider using the zoom function to
identify interesting peaks:</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" tabindex="-1"></a><span class="co"># First find a peak</span></span>
<span id="cb8-2"><a href="#cb8-2" tabindex="-1"></a>out <span class="ot">=</span> <span class="fu">XQTL_zoom</span>(zinc_hanson_pseudoscan, <span class="st">&quot;chr3R&quot;</span>, <span class="dv">13000000</span>, <span class="dv">15000000</span>)</span>
<span id="cb8-3"><a href="#cb8-3" tabindex="-1"></a></span>
<span id="cb8-4"><a href="#cb8-4" tabindex="-1"></a><span class="co"># Then plot the region around the peak</span></span>
<span id="cb8-5"><a href="#cb8-5" tabindex="-1"></a>A1 <span class="ot">=</span> <span class="fu">XQTL_region</span>(zinc_hanson_pseudoscan, out<span class="sc">$</span>chr, out<span class="sc">$</span>start, out<span class="sc">$</span>stop, <span class="st">&quot;Wald_log10p&quot;</span>)</span>
<span id="cb8-6"><a href="#cb8-6" tabindex="-1"></a>A2 <span class="ot">=</span> <span class="fu">XQTL_change_average</span>(zinc_hanson_means, out<span class="sc">$</span>chr, out<span class="sc">$</span>start, out<span class="sc">$</span>stop)</span>
<span id="cb8-7"><a href="#cb8-7" tabindex="-1"></a>A3 <span class="ot">=</span> <span class="fu">XQTL_genes</span>(dm6.ncbiRefSeq.genes, out<span class="sc">$</span>chr, out<span class="sc">$</span>start, out<span class="sc">$</span>stop)</span></code></pre></div>
</div>
<div id="data-format-requirements" class="section level2">
<h2>Data Format Requirements</h2>
<p>Your data should follow these formats:</p>
<ul>
<li><strong>QTL scan data</strong>: Columns <code>chr</code>,
<code>pos</code>, <code>Wald_log10p</code>, and optionally
<code>cM</code></li>
<li><strong>Frequency data</strong>: Columns <code>chr</code>,
<code>pos</code>, <code>TRT</code>, <code>REP</code>,
<code>founder</code>, <code>freq</code></li>
<li><strong>Gene data</strong>: Columns <code>chr</code>,
<code>start</code>, <code>end</code>, <code>gene_name</code>,
<code>strand</code>, <code>is_utr</code></li>
<li><strong>Variant data</strong>: Columns <code>CHROM</code>,
<code>POS</code>, <code>type</code>, <code>subtype</code>, and genotype
columns for each founder</li>
</ul>
</div>
</div>
<div id="references" class="section level1">
<h1>References</h1>
<ul>
<li>NCBI RefSeq: <a href="https://www.ncbi.nlm.nih.gov/refseq/">https://www.ncbi.nlm.nih.gov/refseq/</a></li>
<li>VCF Format Specification: <a href="https://samtools.github.io/hts-specs/VCFv4.2.pdf">https://samtools.github.io/hts-specs/VCFv4.2.pdf</a></li>
</ul>
</div>



<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
